index.html

```diff
@@ -9943,14 +9967,26 @@ <h4 id="Beast Shape - Large Animal">Beast Shape - Large Animal</h4>
                <td>AP Cost
                </td>
             </tr>
+            <tr>
+               <td>Core
+               </td>
+               <td>Savage Bite
+               </td>
+               <td>Make a melee attack against a creature, dealing Heavy+Str damage.
+               </td>
+               <td>E
+               </td>
+               <td>1
+               </td>
+            </tr>
             <tr>
                <td>Core
                </td>
                <td>Barrel
                </td>
                <td>Rush forward smashing all enemies in your path. Move up to 20ft in a
                   straight line and make a regular melee attack against all targets in that line.
-                  All targets hit by the attack take your regular attack damage and are knocked
+                  All targets hit by the attack take Light+Str damage and are knocked
                   <a class="condition" data-tooltip-container>prone</a>. 1 FORT point to resist.
```

Comment:
Same as before, this can't be infinite use. If it is? Idk there's not any clear indication cause there doesn't seem to be a difference between this and the basic attack.